Zend/zend_compile.h

```diff
@@ -382,6 +383,7 @@ typedef struct _zend_class_constant {
 	zval value; /* access flags are stored in reserved: zval.u2.access_flags */
 	zend_string *doc_comment;
 	HashTable *attributes;
+	zend_ast_attr attr;
```

Comment:
const_flags

Reply:
`zval value; /* access flags are stored in reserved: zval.u2.access_flags */` - I'm pretty sure this is where `private|protected|public` flags on class constants are already tracked.

If there is a reason why zval.u2.access_flags could not be reused for tracking ZEND_CLASS_CONST_IS_CASE, document it?